phpcs.xml.dist

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="LiteSpeed_Cache" xsi:noNamespaceSchemaLocation="lsc.xsd">
```

Comment:
Why change from this?
```xml
<ruleset name="LiteSpeed Cache Coding Standards">
```

Reply:
Was added when I tried to fix a bug. But was from somwehere else.
Copied from other repositories.
I reverted the changes.